src/Modules/OrchardCore.Commerce.Payment.Stripe/Drivers/StripeApiSettingsDisplayDriver.cs

```diff
@@ -43,9 +42,12 @@ public class StripeApiSettingsDisplayDriver : SectionDisplayDriver<ISite, Stripe
         _logger = logger;
     }
 
-    public override async Task<IDisplayResult> EditAsync(StripeApiSettings section, BuildEditorContext context)
+    protected override string SettingsGroupId
+        => GroupId;
```

Comment:
Doesn't need to be in a separate line. Please the same for all overrides of `SettingsGroupId`
```suggestion
    protected override string SettingsGroupId => GroupId;
```